default-preset.ttl

```diff
@@ -0,0 +1,18 @@
+@prefix lv2:   <http://lv2plug.in/ns/lv2core#> .
+@prefix pset:  <http://lv2plug.in/ns/ext/presets#> .
+@prefix state: <http://lv2plug.in/ns/ext/state#> .
+
+<default-preset>
+	a pset:Preset ;
+	lv2:appliesTo <http://gareus.org/oss/lv2/xfade> ;
+	lv2:port [
+		lv2:symbol "mode" ;
+		pset:value 0.0
+	] , [
+		lv2:symbol "shape" ;
+		pset:value 0.0
+	] , [
+		lv2:symbol "xfade" ;
+		pset:value 0.0
+	] .
+
```

Comment:
Nope. The defaults are already in `xfade.ttl.in`. I will not maintain a preset for defaults.